### PR TITLE
fixup! Missing LINODE_TOKEN error message

### PR DESCRIPTION
**Pull request by cbzzz**
# Description
Addresses: https://github.com/linode/cluster-api-provider-linode/issues/76#issue-2111484116

# Notes
I'm not sure we should be using a global `LINODE_TOKEN` for the operator (and all it's managed workload clusters) here. We should allow workload clusters to specify the Linode account in which to provision its resources.

### CODE REVIEW

**Review by mhmxs — APPROVED** [review]
It makes sense, lots of other project uses `setupLog` in the same way.

**Comment by eljohnson92** [comment]
@mhmxs is it expected that the e2e-test failure just because the test is coming from a fork?

**Comment by cbzzz** [comment]
@mhmxs It seems like the E2E test doesn't have the LINODE_TOKEN environment variable set. Is this expected? Thanks!

https://github.com/linode/cluster-api-provider-linode/actions/runs/7805739210/job/21292435053#step:7:346

**Comment by AshleyDumaine** [comment]
@cbzzz I think you would need to set the LINODE_TOKEN secret on your fork for GHA to pass: https://github.com/linode/cluster-api-provider-linode/blob/main/.github/workflows/build_test_ci.yml#L139C35-L139C47

**Comment by mhmxs** [comment]
> Is this expected?

@cbzzz I our current setup: Yes

**Comment by mhmxs** [comment]
> set the LINODE_TOKEN

@cbzzz @AshleyDumaine keep in mind, that test execution should cost money. At the time of this we don't have any tests, but this should change soon: https://github.com/linode/cluster-api-provider-linode/pull/80

**Comment by eljohnson92** [comment]
I think there is a larger discussion about when we want to run these e2e tests but I don't think that should be blocking on this PR. I'm good to merge as is if there are no other comments

**Review by eljohnson92 — APPROVED** [review]

**Review by AshleyDumaine — APPROVED** [review]

**Pull request merged**
